__tests__/app.test.js

```diff
@@ -11,7 +11,16 @@ beforeEach(() => {
 afterAll(() => {
     return db.end();
 });
-
+describe('hits the /api endoint', () => {
+    test('should return a 200 status code', () => {
+        return request(app).get('/api').expect(200);
+    });
+    test("should return an object describing all available endpoints", () => {
+        return request(app).get('/api').then(({body}) => {
+            expect(body).toBe(body)
```

Comment:
Instead of asserting body to be body as this will always pass.
We can require in the endpoints.json and make sure the response looks like that required json.